Cars/CarsServices/Controllers/HomeController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CarsServices.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
+        }
+       
+        public ActionResult StartPage()
+        {
+            return View();
```

Comment:
Start page is empry, why? Give users some content, forexamplea cool duck in sunglasses with a car